src/index.js

```diff
@@ -0,0 +1,139 @@
+  tempLabel.textContent = `${state.temperature}°F`;
+  const changeLandscape = document.getElementById('landscape');
+
+  let landscape = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  if (state.temperature >= 80) {
+    tempLabel.className = 'red';
+    landscape = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (state.temperature >= 70) {
+    tempLabel.className = 'orange';
+    landscape = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (state.temperature >= 60) {
+    tempLabel.className = 'yellow';
+    landscape = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (state.temperature >= 50) {
+    tempLabel.className = 'green';
+    landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (state.temperature <= 49) {
+    tempLabel.className = 'teal';
+    landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+  console.log(tempLabel.className);
+  changeLandscape.textContent = `${landscape}`;
+};
+
+const increaseTemp = () => {
+  state.temperature += 1;
+  updateTempLabel();
+  console.log('increase temp');
+};
+
+const decreaseTemp = () => {
+  state.temperature -= 1;
+  updateTempLabel();
+  console.log('decrease temp');
+};
+
+const updateCityLabel = () => {
+  const citySearch = document.getElementById('cityLabel');
+  citySearch.textContent = `${state.city}`;
+};
+
+const updateCityWithTemp = () => {
+  updateCity();
+  getLatAndLong();
+};
+
+const updateCity = () => {
+  const searchFieldInput = document.getElementById('citySearch');
+  state.city = searchFieldInput.value;
+  updateCityLabel();
+};
+
+const updateTheSky = () => {
+  const skyLabel = document.querySelector('#sky-types').value;
+  const changeSky = document.getElementById('sky-horizon');
+
+  let sky = '☁️ ☁️ ☁️🌞☁️ ☁️ ☁️';
+  if (skyLabel === 'sunny') {
+    sky = '☁️ ☁️ ☁️🌞☁️ ☁️ ☁️';
+  } else if (skyLabel === 'cloudy') {
+    sky = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (skyLabel === 'rainy') {
+    sky = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (skyLabel === 'snowy') {
+    sky = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  }
+  changeSky.textContent = `${sky}`;
+};
+
+const resetCity = () => {
+  const cityName = document.getElementById('citySearch');
+  cityName.value = '';
+  state.city = 'Honolulu';
+  updateCityLabel();
+};
+
+const registerEventHandlers = () => {
+  const tempUpButton = document.getElementById('tempUp');
+  tempUpButton.addEventListener('click', increaseTemp);
+  const tempDownButton = document.getElementById('tempDown');
+  tempDownButton.addEventListener('click', decreaseTemp);
+  const submitCityButton = document.getElementById('submitCity');
+  submitCityButton.addEventListener('click', updateCityWithTemp);
+  const selectItem = document.getElementById('sky-types');
+  selectItem.addEventListener('change', updateTheSky);
+  const resetButton = document.getElementById('reset');
+  resetButton.addEventListener('click', resetCity);
+  const citySearchEl = document.getElementById('citySearch');
+  citySearchEl.addEventListener('input', updateCity);
+};
+
+document.addEventListener('DOMContentLoaded', registerEventHandlers);
+
+const getLatAndLong = () => {
```

Comment:
small note. I would move this function and the function below this before you register the event handlers. Just to make the code more readable.